<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MBI_GUI</class>
 <widget class="QMainWindow" name="MBI_GUI">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1920</width>
    <height>1080</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MBI_GUI</string>
  </property>
  <property name="windowOpacity">
   <double>1.000000000000000</double>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="LoadBitFile">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>30</y>
      <width>311</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Load Bit File</string>
    </property>
   </widget>
   <widget class="QPushButton" name="LoadPattFile">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>130</y>
      <width>311</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Load Camera Pattern File</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="Exposure">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>230</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="text">
     <string>1</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="exposurelabel">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>230</y>
      <width>221</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Exposure Time [us]</string>
    </property>
   </widget>
   <widget class="QLabel" name="maskslabel">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>290</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Masks</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="Masks">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>290</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="text">
     <string>600</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="MaskChanges">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>350</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="maskschangeslabel">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>350</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Mask Changes</string>
    </property>
   </widget>
   <widget class="QLabel" name="maskschangeslabel_2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>410</y>
      <width>231</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Subscenes per Change</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="SubChange">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>410</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="DispImage">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>470</y>
      <width>181</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Display Image</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QComboBox" name="DispFrame">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>490</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
    <item>
     <property name="text">
      <string>ALL</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>CEP</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>CEP-TOF</string>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="RecVideo">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>570</y>
      <width>311</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Record Video</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="SaveImages">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>670</y>
      <width>141</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Save Images</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="NumImages">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>700</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>1</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>670</y>
      <width>101</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Number of Images</string>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="TestPixels">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>770</y>
      <width>311</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>Test Pixels</string>
    </property>
   </widget>
   <widget class="MPLWidget" name="mplimg1" native="true">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>30</y>
      <width>750</width>
      <height>700</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="MPLWidget" name="mplimg2" native="true">
    <property name="geometry">
     <rect>
      <x>1140</x>
      <y>30</y>
      <width>750</width>
      <height>700</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="MPLWidget" name="mplhist1" native="true">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>750</y>
      <width>751</width>
      <height>301</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="MPLWidget" name="mplhist2" native="true">
    <property name="geometry">
     <rect>
      <x>1140</x>
      <y>750</y>
      <width>751</width>
      <height>301</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MPLWidget</class>
   <extends>QWidget</extends>
   <header location="global">mplwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
